frontend/amundsen_application/static/js/components/Table/index.tsx

```diff
@@ -3,6 +3,8 @@
 
 import * as React from 'react';
 
+import { FormattedDataType } from 'features/ColumnList';
```

Comment:
Conceptually, this component shouldn't export anything from the features folder.
Would it be possible to make this type something that the table exposes instead?